pennylane/transforms/add_noise.py

```diff
@@ -52,10 +56,10 @@ def add_noise(tape, noise_model, level=None):
 
     .. note::
 
-        For a given ``model_map`` within a ``NoiseModel``, if multiple conditionals in the ``model_map``
-        evaluate to ``True`` for an operation, then the noise operations defined via their respective
-        noisy quantum functions will be added in the same order in which the conditionals appear in the
-        ``model_map``.
+        For a given ``model_map`` and ``meas_map`` within a ``NoiseModel``, if multiple conditionals in the
```

Comment:
This is probably correct but I'm surprised that adding noise with respect to an observable is done the same as with respect to a gate. Do other frameworks do it this way too?

Reply:
To clarify, we are doing something like this:

$\langle \phi| \text{Noise}^{\dagger} \mathcal{O}  \text{Noise} |\phi \rangle$

it is perhaps strange that the noise is "symmetric" in the Observable, since the noise is not supposed to have any property with respect to the observable